.changeset/upset-paths-stand.md

```diff
@@ -2,6 +2,6 @@
 "@fluidframework/container-loader": major
 ---
 
-Container caching in the Loader is removed
+container-loader: Container caching in the Loader is removed
```

Comment:
@alexvy86 - Can you follow up with stakeholders on FF/Loop here to make sure this won't end up blocking the bump?  It sounded like the migration was a little disjointed based on the firedrill that happened during the last major bump.